src/main/java/com/gregtechceu/gtceu/client/renderer/GTRenderTypes.java

```diff
@@ -0,0 +1,29 @@
+package com.gregtechceu.gtceu.client.renderer;
+
+import net.minecraft.client.renderer.RenderStateShard;
+import net.minecraft.client.renderer.RenderType;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
+
+import com.mojang.blaze3d.vertex.DefaultVertexFormat;
+import com.mojang.blaze3d.vertex.VertexFormat;
+
+@OnlyIn(Dist.CLIENT)
+public class GTRenderTypes extends RenderType {
```

Comment:
you probably don't need to extend RenderType at all.

Reply:
In fact, this is necessary because the access level of some required fields is protected, which is also the practice of most mods